.travis.yml

```diff
@@ -82,12 +87,17 @@ after_success:
  - ccache -s
  - test "${TRAVIS_BUILD_STAGE_NAME}" = "Test" || travis_terminate 0
  - if [[ ${COVERAGE} ]]; then
-     pushd build && wget -O codecov https://codecov.io/bash && chmod +x codecov &&
+     pushd build &&
      if [[ ${CC} = clang ]]; then
-       ./codecov -x "llvm-cov gcov" -F "${CC}";
+       codecov -x "llvm-cov gcov" -F "${CC}";
      else
-       ./codecov -F "${CC}";
+       codecov -F "${CC}";
      fi &&
+     if [[ $(doxygen --version) != 1.8.16 ]]; then
+       echo "Patching Doxygen files (see doxygen/doxygen#6892)" &&
+       sed -e '/enable_if</s/&/\&amp;/g' -e '/enable_if/s/<\([^a-z/]\)/\&lt;\1/g' -e '/enable_if/s/ >/ \&gt;/g' -i xml/*.xml;
```

Comment:
Here it looks like you are removing the offending code in the XML files? Is 1.8.16 the latest doxygen version that has the bug fix?

Reply:
Yes, the fix is part of the next release, 1.8.16.

Reply:
should we use `<` instead of `!=` for the version check assuming that future versions will also not have this bug?